src/screens/blockchain/near/recover-account/recover-account.tsx

```diff
@@ -241,6 +263,25 @@ export class RecoverNearAccountComponent extends React.Component<
                     }
                 } else {
                     // Input is valid
+                    // Search for lockup id on the owner
+                    const lockupId = accountInstance.getLockupContract(
+                        accountId,
+                        String(this.props.chainId)
```

Comment:
this wouldn't be needed if you transform chanId parameter from string to ChainIdType in `getLockupContract`